libafl/src/feedbacks/mod.rs

```diff
@@ -940,7 +940,7 @@ where
         observers: &OT,
         testcase: &mut Testcase<I>,
     ) -> Result<(), Error> {
-        let observer = observers.get(&self.observer_handle).unwrap();
+        let observer = observers.get(&self.observer_handle).expect("Observer referenced by TimeFeedback is not found in observers given to the fuzzer");
```

Comment:
I think the right way here would be to return `Err(Error::illegal_state("Observer referenced by TimeFeedback is not found in observers given to the fuzzer"))`

Reply:
And anyway this should never happen, how did you reach this state?

Reply:
I reached it because I didn't give the fuzzer all my observers. I got confused when it just hit the unwrap, so then I made the PR.